scene/animation/animation_tree.cpp

```diff
@@ -967,6 +967,7 @@ void AnimationTree::_process_graph(double p_delta) {
 			int pingponged = as.pingponged;
 #ifndef _3D_DISABLED
 			bool backward = signbit(delta);
+			bool calc_root = !seeked || as.seek_root;
```

Comment:
AnimationTree performs a seek to change the playback time, but should be able to choose whether to calculate the root motion or not.